integration_test/pg/constraints_test.exs

```diff
@@ -9,16 +9,16 @@ defmodule Ecto.Integration.ConstraintsTest do
   defmodule ConstraintMigration do
     use Ecto.Migration
 
-    @table table(:constraints_test)
+    @table table(:constraints_test, comment: "table")
```

Comment:
I would move those comments to the actual migration file in `integration_test/support/migrations.exs` since then it could be useful for all databases that support migrations. :)